### ISSUE TITLE
Conftest error after VM reconnects

### ISSUE THREAD

**Issue by HariniMohan0102** [tenstorrent/tt-metal#14949]
**Describe the bug**
Whenever VM (172.27.45.41) reconnects the build gets corrupetd and I get conftest error:
```
ConftestImportFailure: ModuleNotFoundError: No module named 'tests' (from /home/ubuntu/harini/tt-metal/conftest.py)
For more information see https://pluggy.readthedocs.io/en/stable/api_reference.html#pluggy.PluggyTeardownRaisedWarning
  config = pluginmanager.hook.pytest_cmdline_parse(
ImportError while loading conftest '/home/ubuntu/harini/tt-metal/conftest.py'.
conftest.py:24: in <module>
    from tests.scripts.common import run_process_and_get_result
E   ModuleNotFoundError: No module named 'tests'
```
While trying to do a clean build, I get build error:
```
../tt_metal/llrt/tt_cluster.hpp:19:10: fatal error: 'noc/noc_parameters.h' file not found
   19 | #include "noc/noc_parameters.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~
1 error generated.
[339/977] Building CXX object tt_metal/third_party/umd/device/CMakeFiles/device.dir/tt_silicon_driver.cpp.o
ninja: build stopped: subcommand failed.
```
Already tried the workarounds such as installing ninja in build folder, deleting the build related folders and doing a clean build again and again, but nothing works.

**_Note_**: The tests were running good before the VM got disconnected. After connecting the issue arises. Moving to a new repo and a fresh build works, but once the VM disconnects, the same issue comes again.

**To Reproduce**
Check with VM: 172.27.45.41 (ubuntu@mcw-model-n150-3)

**Expected behavior**
Expected to resolve the build issue.

**Please complete the following environment information:**
- OS: [e.g. Ubuntu 20.04]
- Device: WH n150

**Comment by tt-rkim** [comment]
I just logged into your VM:

I ran and built with `export PYTHONPATH=$(pwd) && export ARCH_NAME=wormhole_b0` and just with `build_metal.sh` got up to:

```
[821/958] Building CXX object ttnn/CMakeFiles/ttnn.dir/cpp/ttnn/tensor/tensor_ops.cpp.o
```

on this VM, which I'm assuming is past your point.

Are you building correctly?

**Comment by HariniMohan0102** [comment]
@tt-rkim, the following are the build commands used for a fresh build:
```
rm -rf build build_Release
git submodule foreach 'git lfs fetch --all && git lfs pull'
git submodule update --init --recursive
export ARCH_NAME=wormhole_b0
export TT_METAL_HOME=$(pwd)	
export PYTHONPATH=$(pwd)
export TT_METAL_ENV=dev
./build_metal.sh -p
./create_venv.sh
source python_env/bin/activate
```
When VM is disconnected and connected again, I was unable to build successfully in that repo for a while (due to errors I reported). After a while, it works again (as it did for you). In the meantime, I had to clone fresh repo and work on that repo (where build runs smoothly).

I am not sure, why this inconsistency on the repo, when VM disconnects. I'll let you know, if I observe this consistently. 
Thank you
